modes/longitudinal/src/index.ts

```diff
@@ -83,57 +39,15 @@ function modeFactory({ modeConfiguration }) {
      */
     onModeEnter: function ({ servicesManager, extensionManager, commandsManager }: withAppTypes) {
       const { measurementService, toolbarService, toolGroupService } = servicesManager.services;
-
       measurementService.clearMeasurements();
 
       // Init Default and SR ToolGroups
-      initToolGroups(extensionManager, toolGroupService, commandsManager);
-
-      toolbarService.addButtons(toolbarButtons);
-      toolbarService.createButtonSection('primary', [
-        'MeasurementTools',
-        'Zoom',
-        'Pan',
-        'TrackballRotate',
-        'WindowLevel',
-        'Capture',
-        'Layout',
-        'Crosshairs',
-        'MoreTools',
-      ]);
-
-      toolbarService.createButtonSection('measurementSection', [
-        'Length',
-        'Bidirectional',
-        'ArrowAnnotate',
-        'EllipticalROI',
-        'RectangleROI',
-        'CircleROI',
-        'PlanarFreehandROI',
-        'SplineROI',
-        'LivewireContour',
-      ]);
+      mode.initToolGroups(extensionManager, toolGroupService, commandsManager);
 
-      toolbarService.createButtonSection('moreToolsSection', [
-        'Reset',
-        'rotate-right',
-        'flipHorizontal',
-        'ImageSliceSync',
-        'ReferenceLines',
-        'ImageOverlayViewer',
-        'StackScroll',
-        'invert',
-        'Probe',
-        'Cine',
-        'Angle',
-        'CobbAngle',
-        'Magnify',
-        'CalibrationLine',
-        'TagBrowser',
-        'AdvancedMagnify',
-        'UltrasoundDirectionalTool',
-        'WindowLevelRegion',
-      ]);
+      toolbarService.addButtons(getCustomization('toolbarButtons'));
```

Comment:
@pedrokohler - to customize the toolbar, you will also need these two lines, and to create a new custom section that extends the default tools/buttons.